### PR TITLE
added 5 tasks for Homework 7

### PR DESCRIPTION
**Pull request by vithaiduchyk**

### CODE REVIEW

File: Homework_7/Task_3.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+for i in range(len(numbers)):
+    for j in range(i + 1, len(numbers)):
+        if numbers[i] == numbers[j]:
+            all_unique = False
+            break
+    if not all_unique:
+        break
+
+if all_unique:
```
Comment (MarinaZelenska) [inline Homework_7/Task_3.py]: Can be better)
image

File: Homework_7/Task_4.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+elements = [1, 5, 68, 0]
+min_value = None
+
+for element in elements:
+    if min_value is None:
+        min_value = element
+    elif element < min_value:
```
Comment (MarinaZelenska) [inline Homework_7/Task_4.py:7]: else better

File: Homework_7/Task_4.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+        min_value = element
+
+print(f"Min value in list is: {min_value}")
```
Comment (MarinaZelenska) [inline Homework_7/Task_4.py:8]: Ca be better)
image

File: Homework_7/Task_3.py (hunk 1 above)
Comment (vithaiduchyk) [reply Homework_7/Task_3.py]: I'll fix it

File: Homework_7/Task_4.py (hunk 3 above)
Comment (vithaiduchyk) [reply Homework_7/Task_4.py:8]: I'll fix it

**Pull request merged**